apps/website/app/components/TeamPerson.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+import Image from "next/image";
+import Link from "next/link";
+
+export interface TeamMember {
+  name: string;
+  title: string;
+  image: string;
+  links?: {
+    twitter?: string;
+    github?: string;
+    website?: string;
+  };
+}
+
+export function TeamPerson({ member }: { member: TeamMember }) {
+  return (
+    <div className="flex flex-col items-center space-y-4 text-center">
+      <div className="relative h-40 w-40 overflow-hidden rounded-full">
+        <Image
+          src={member.image}
+          alt={member.name}
+          fill
+          className="object-cover"
+        />
+      </div>
+      <div>
+        <h3 className="text-xl font-semibold text-neutral-dark">
+          {member.name}
+        </h3>
+        <p className="text-neutral-dark/80">{member.title}</p>
+      </div>
+      {member.links && (
```

Comment:
Links is a nice touch in case we want to add it later.  I'm fine with leaving it, but the links to the images are broken.
We should either fix the links and add the images, leave a comment noting it is for future use, or remove it altogether.

We'll bring it up in the next all hands to see what the team thinks of adding them:
https://roamresearch.com/#/app/discourse-graphs/page/ENJzRTJ0m

Reply:
i fixed the social image links